app/src/main/kotlin/org/fossify/musicplayer/fragments/AlbumsFragment.kt

```diff
@@ -80,7 +81,9 @@ class AlbumsFragment(context: Context, attributeSet: AttributeSet) : MyViewPager
     }
 
     override fun onSearchQueryChanged(text: String) {
-        val filtered = albums.filter { it.title.contains(text, true) }.toMutableList() as ArrayList<Album>
+        val normalizedText = text.normalizeString()
+        val filtered = albums.filter { it.title.normalizeString().contains(normalizedText, true)
+        }.toMutableList() as ArrayList<Album>
```

Comment:
```suggestion
        val filtered = albums.filter {
            it.title.normalizeString().contains(normalizedText, true)
        }.toMutableList() as ArrayList<Album>
```

Same problem in other fragments. You can do targeted formatting by selecting the block of code you want to format in the IDE.